test/e2e/kubectl.go

```diff
@@ -298,7 +298,11 @@ var _ = framework.KubeDescribe("Kubectl client", func() {
 
 			// Build the static kubectl
 			By("Finding a static kubectl for upload")
-			testStaticKubectlPath, err := findBinary("kubectl", "linux/386")
+			targetBinary := "linux/386"
```

Comment:
@luxas If I understand correctly, you meant it is safe to just use amd64 for all cases?